src/Ballot.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace PivotLibre\Tideman;
 
```

Comment:
Switching this was the fastest way to implement this, not necessarily the best way. Some consequent usage updates are in this PR.